### ISSUE TITLE
favicon is lagging

### ISSUE THREAD

**Issue by suessmann** [anakinsk/SafariStand#55]
Hey! 

No idea whether it is an important issue for now, but what I spot using SafariStand on High Sierra. If you click on a link, say, in your personal messages in vk.com, then the favicon of the tab will be a favicon of vk.com/redirect, but not a favicon of an actual website I am on. See screenshots for details.

Anyway, thanks for releasing such a helpful thing! 

<img width="151" alt="screen shot 2017-10-08 at 19 34 56" src="https://user-images.githubusercontent.com/26511522/31318777-df441a72-ac5f-11e7-9305-c68ba6ed3034.png">
<img width="160" alt="screen shot 2017-10-08 at 19 34 50" src="https://user-images.githubusercontent.com/26511522/31318776-df4350f6-ac5f-11e7-8bd1-9224e82f75c1.png">

Btw, I noticed that it happens only if I open a link in a new tab.

**Comment by anakinsk** [comment]
but then after the actual page is loaded the right favicon is displayed, is that correct?

**Comment by staniki** [comment]
@anakinsk Вань, а то что фавиконы теперь немного тусклые (не такие яркие как раньше) и немного размытые, это нормально?

**Comment by anakinsk** [comment]
@staniki хрен его знает. Я честно не заметил что они стали хуже качеством, надо будет глянуть что у нас там с размерами. Насчет того что они стали немного тусклее, я думаю что это сафари накладывает какой то градиент поверх всей кнопки, который и влияет на это все. Фиксить это не буду сразу говорю 😀 Это фича, так больше гармонирует с самим серым интерфейсом сафари)

**Comment by suessmann** [comment]
@anakinsk not really! I need to reload the page manually, only then the true favicon appears! Btw, the same thing with google – after I open a link in a new tab, it has a favicon of google, but not of the website I am actually on.

**Comment by suessmann** [comment]
May it be the problem of mine or anyone else has the same issue?

**Comment by anakinsk** [comment]
No I think you're right. I can see that behaviour with google. I will look into it, good catch!
